src/views/Profile/ProfileCreation/Mint.tsx

```diff
@@ -54,6 +59,7 @@ const Mint = () => {
           <NftSelectionCard nft={nft1} isChecked />
         </CardBody>
       </Card>
+      <NextStepButton onClick={nextStep}>{TranslateString(999, 'Next Step')}</NextStepButton>
```

Comment:
I moved the next step button to a per-step basis because it seems each step will need to handle the logic on whether the user can progress to the next one.